pkg/hs/urbit-king/lib/Urbit/Arvo/Effect.hs

```diff
@@ -174,19 +175,22 @@ instance FromNoun Deco where
 
 instance ToNoun Tint where
   toNoun = \case
-    TintR    -> toNoun $ Cord "r"
-    TintG    -> toNoun $ Cord "g"
-    TintB    -> toNoun $ Cord "b"
-    TintC    -> toNoun $ Cord "c"
-    TintM    -> toNoun $ Cord "m"
-    TintY    -> toNoun $ Cord "y"
-    TintK    -> toNoun $ Cord "k"
-    TintW    -> toNoun $ Cord "w"
-    TintNull -> Atom 0
+    TintR          -> toNoun $ Cord "r"
+    TintG          -> toNoun $ Cord "g"
+    TintB          -> toNoun $ Cord "b"
+    TintC          -> toNoun $ Cord "c"
+    TintM          -> toNoun $ Cord "m"
+    TintY          -> toNoun $ Cord "y"
+    TintK          -> toNoun $ Cord "k"
+    TintW          -> toNoun $ Cord "w"
+    TintNull       -> Atom 0
+    TintTrue r g b -> Cell (Atom r) $ Cell (Atom g) (Atom b)
```

Comment:
If you like, you can use C and A instead of Cell and Atom here.

Reply:
pls tell me there's some way in haskell to say "TintR means `r`, TintG means `g`…" that doesn't use a line per character

Reply:
I'm _assuming_ you can't just do a hoon/ ts-ish `Tint ("r" | "g" | "b" | "c" | "m" | "y" | "k" | "w")` type that would render the point moot?

Reply:
Unfortunately, no to both of these, honorable Anton.